radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -1112,9 +1114,10 @@ private Result(
 						final JsonObject errorData = data == null ? null : data.getAsJsonObject();
 						final ActionExecutionExceptionBuilder exceptionBuilder = new ActionExecutionExceptionBuilder()
 							.errorData(errorData);
+						final Consumer<ActionExecutionExceptionReason> addReason = exceptionBuilder::addReason;
```

Comment:
Why this change?

Reply:
Code as is does not pass compilation:

/home/siy/Projects/Radix/radixdlt-java/radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java:1117: error: incompatible types: invalid method reference
                                                        .forEach(exceptionBuilder::addReason);
                                                                 ^
    incompatible types: Object cannot be converted to ActionExecutionExceptionReason